.npmignore

```diff
@@ -1,25 +1,9 @@
-.project
 .settings
-
-demos
-examples
-Gruntfile.js
-products
-
-src/components
-src/lib/fonts
-src/lib/jquery/plugins
-src/lib/jquery/ui
-
-src/framework/core/css
-src/framework/preferences
-
-tests/3rd-party-tests
-tests/component-tests
-tests/framework-tests
-tests/manual-tests
-tests/all-tests.html
-tests/lib/jquery-ui
-tests/lib/mockjax
-tests/lib/README
-tests/test-core/utils/js/DebugFocus.js
+.gitignore
+.jshintignore
+.jshintrc
+.npmignore
+.project
+demos/
+examples/
+tests/
```

Comment:
Did we want to leave in the testing infrastructure and just remove the actual tests. This seems to wipeout all of the tests including the ones specified in [devIncludes.json](https://github.com/fluid-project/infusion/blob/master/src/module/devIncludes.json)